lib/configuration_parser.rb

```diff
@@ -21,13 +25,18 @@ def parse(options)
 
     def parse_1_0(config, dist_folder, bootstrap)
       if config.include? 'deploy'
+        abort "only 'bootstrap' or 'recurring are allowed for deploy.fore in .deploy-now/config.yaml" unless config['deploy']['force'].eql?('bootstrap') || config['deploy']['force'].eql?('recurring') || config['deploy']['force'].nil?
```

Comment:
There is a typo. Should be `deploy.force` not `deploy.fore`

Reply:
fixed